backend/src/settings.py

```diff
@@ -0,0 +1,4 @@
+origins = [
```

Comment:
What is this file for out of curiosity?

Reply:
This is where we store data that is relevant to the backend instance such as acceptable origins.